README.md

```diff
@@ -86,6 +86,19 @@ type Options struct {
 	// If given, copy.Copy refers to this fs.FS instead of the OS filesystem.
 	// e.g., You can use embed.FS to copy files from embedded filesystem.
 	FS fs.FS
+
+	// NumOfWorkers represents the number of workers used for
+	// concurrent copying contents of directories.
+	// If 0 or 1, it does not use goroutine for copying directories.
+	// Please refer to https://pkg.go.dev/golang.org/x/sync/semaphore for more details.
+	NumOfWorkers int64
```

Comment:
Any reason not to make this just an `int`?

Reply:
According to semaphore.NewWeighted